src/socketio.quiver.json

```diff
@@ -61,7 +61,7 @@
 
       "acceptUserInvitation": {
        "type": "method",
-        "value": ["string"]
+        "value": ["string", "string"]
```

Comment:
I'd rather leave this as a string for now since uProxy is only going to be sending a string, plus I think this data should always be encrypted.